web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.jsx

```diff
@@ -129,22 +131,28 @@ export const MachineLearningSettings = () => {
         />
 
         {backends.length > 0 && (
-          <>
-            <Description>
+          <div className="my-wide">
+            <Typography size="small" className="text-neutral-content-subtler">
               A connected model has been detected! If you wish to fetch predictions from this model, please follow these
               steps:
-              <br />
-              <br />
-              1. Navigate to the <i>Data Manager</i>.<br />
+            </Typography>
+            <Typography size="small" className="text-neutral-content-subtler mt-base">
+              1. Navigate to the <i>Data Manager</i>.
+            </Typography>
+            <Typography size="small" className="text-neutral-content-subtler mt-tighter">
               2. Select the desired tasks.
-              <br />
-              3. Click on <i>Retrieve predictions</i> from the <i>Actions</i> menu.
-            </Description>
-            <Description>
+            </Typography>
+            <Typography size="small" className="text-neutral-content-subtler mt-tighter">
+              3. Click on <i>Batch predictions</i> from the <i>Actions</i> menu.
```

Comment:
Would be more semantic to have this collapsed to an `ol > li`

Reply:
Good point, thanks. I thought of that but tailwind base resets ordered lists and this will require digging deeper into disabling the reset for these lists or create another component and didn't want to increase the scope.